modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1930,5 +1930,183 @@ PERF_TEST_P_(ResizeFxFyPerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+PERF_TEST_P_(ParseSSDWLPerfTest, TestPerformance)
+{
+    cv::Size sz                   = get<0>(GetParam());
+    float confidence_threshold    = get<1>(GetParam());
+    int filter_label              = get<2>(GetParam());
+    cv::GCompileArgs compile_args = get<3>(GetParam());
```

Comment:
I'd recommend to use `std::tie()` - at least it will fail the compilation if the Parameters structure is extended but your code is not updated.

Reply:
Fixed.